modules/reference/pages/topic-properties.adoc

```diff
@@ -151,6 +154,7 @@ Configure properties to manage the disk space used by a topic:
 - Retain logs for a maximum duration before cleanup (<<retentionms, `retention.ms`>>).
 - Periodically close an active log segment (<<segmentms, `segment.ms`>>).
 - Limit the maximum size of an active log segment (<<segmentbytes, `segment.bytes`>>).
+- Cache batches until the segment appender chunk is full, instead of flushing for every `acks=all` write (<<writecaching,`write.caching`>>). With `write.caching` enabled, flushes follow <<flushms, `flush.ms`>> and <<flushbytes, `flush.bytes`>>, whichever is reached first. 
```

Comment:
Lets remove all references to "Segment appender chunk"..the RFC content was geared towards core engineers who knew the internals.. this is not a user facing construct.